src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/noenumtrailingcomma/InputNoEnumTrailingCommaOne.java

```diff
@@ -0,0 +1,96 @@
+/*
+NoEnumTrailingComma
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.noenumtrailingcomma;
+
+public class InputNoEnumTrailingCommaOne {
+
+    enum Foo1 {
+        FOO,
+        BAR; // ok
```

Comment:
given the CI logs, it says 
`/home/vsts/work/1/s/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/noenumtrailingcomma/InputNoEnumTrailingCommaOne.java:13: // ok comments should have an explanation, or they are unnecessary. [UnnecessaryOkComment]`

Refer: https://dev.azure.com/romanivanovjr/a35b6275-caac-4633-b371-79a5ab0f70c1/_apis/build/builds/18229/logs/171

To my understanding, it needs to add description after ok. Something like, `// ok, trailing comma not present after last constant in enum definition`

Refer the docs here for understanding of the check and maybe you can frame a better statement. 
https://checkstyle.org/checks/coding/noenumtrailingcomma.html

And, after adding some explanation to ok statements, you can try running in your local to see if the failing test test cases are gone.

Reply:
I got it but there is an another issue which is about removing these `// ok` comments. Here's the link of that issue: [link](https://github.com/checkstyle/checkstyle/issues/13213)

So I guess this doesn't need to add - explanation to the `// ok` comments. Infact I made a PR regarding removal of `// ok` comments for the file I'm currently working in this issue. Here's the link of that PR: [link](https://github.com/checkstyle/checkstyle/pull/14130). The changes are approved by Mr.Roman but the PR is not merged yet. Mr.Roman said I can start working on other issues until [this]( https://github.com/checkstyle/checkstyle/pull/14130 ) PR gets merged.